pinot-core/src/main/java/org/apache/pinot/core/util/OsCheck.java

```diff
@@ -46,9 +48,11 @@ public enum OSType {
     Windows, MacOS, Linux, Other
   }
 
+  private static final Logger log = LoggerFactory.getLogger(OsCheck.class);
```

Comment:
(convention) Use all uppercase for static final fields, suggest also fixing `_detectedOS` to `DETECTED_OS` or `OS_TYPE`
```suggestion
  private static final Logger LOGGER = LoggerFactory.getLogger(OsCheck.class);
```